cinema/serializers.py

```diff
@@ -0,0 +1,18 @@
+from rest_framework import serializers
+
+from cinema.models import Movie
+
+
+class MovieSerializer(serializers.Serializer):
+    id = serializers.IntegerField(read_only=True)
+    title = serializers.CharField(required=True)
+    description = serializers.CharField(required=True)
```

Comment:
The same applies to this line, you don't need to explicitly set `required=True`.